### ISSUE TITLE
seemingly random loss of mysql connection

### ISSUE THREAD

**Issue by damian0604** [ccs-amsterdam/3bij3#10]
occured after the intake questionnaire

3bij3.log:

```
Traceback (most recent call last):
  File "/home/damian/github/3bij3/venv/lib/python3.8/site-packages/flask/app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/damian/github/3bij3/venv/lib/python3.8/site-packages/flask/app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/damian/github/3bij3/venv/lib/python3.8/site-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/damian/github/3bij3/venv/lib/python3.8/site-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/damian/github/3bij3/venv/lib/python3.8/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
  File "/home/damian/github/3bij3/app/blueprints/multilingual/routes.py", line 217, in newspage
    cursor = connection.cursor(buffered=True)
  File "/home/damian/github/3bij3/venv/lib/python3.8/site-packages/mysql/connector/connection_cext.py", line 632, in cursor
    raise OperationalError("MySQL Connection not available.")
mysql.connector.errors.OperationalError: MySQL Connection not available.
```

**Comment by damian0604** [comment]
It may (may) have something to do with the docker deployment, even though the number of connections etc. should actually be fine. Testing it now with non-docker mysql on ubuntu instead...

**Comment by Rptkiddle** [comment]
Possibly related: I have been getting a 500 (internal server) error in browser. This has seemed to increase in frequency (on navigation actions) since the changes to the backend were implemented, today I cannot reach the site at all.

**Comment by damian0604** [comment]
Probably fixed by #14 - reopen if problem reoccurs
